torch/nn/intrinsic/qat/modules/conv_fused.py

```diff
@@ -96,17 +96,18 @@ def _forward(self, input):
 
         if self.training and not self.freeze_bn:
             # recovering original conv to get original batch_mean and batch_var
-            conv_orig = conv / scale_factor.reshape([1, -1, 1, 1])
+            if self.bias is not None:
+                conv_orig = conv / scale_factor.reshape([1, -1, 1, 1]) + self.bias.reshape([1, -1, 1, 1])
+            else:
+                conv_orig = conv / scale_factor.reshape([1, -1, 1, 1])
             batch_mean = torch.mean(conv_orig, dim=[0, 2, 3])
             batch_var = torch.var(conv_orig, dim=[0, 2, 3], unbiased=False)
             n = float(conv_orig.numel() / conv_orig.size()[1])
             unbiased_batch_var = batch_var * (n / (n - 1))
             batch_rstd = torch.ones_like(batch_var, memory_format=torch.contiguous_format) / torch.sqrt(batch_var + self.eps)
 
-            rescale_factor = running_std * batch_rstd
-            conv = conv * rescale_factor.reshape([1, -1, 1, 1])
-            conv = conv + (self.beta - self.gamma * batch_mean * batch_rstd).reshape([1, -1, 1, 1])
-
+            conv = (self.gamma * batch_rstd).reshape([1, -1, 1, 1]) * conv_orig + \
```

Comment:
The correct  equation is 
``` python
conv = (self.gamma * batch_rstd).reshape([1, -1, 1, 1]) * (conv_orig + self.bias -  batch_mean).reshape([1, -1, 1, 1])  + self.beta.reshape([1, -1, 1,1])
```

Reply:
This would give incorrect results as batch_mean and batch_var need to take the bias into account